src/DropMenu.js

```diff
@@ -58,15 +87,23 @@ class DropMenu extends PureComponent {
     }
 }
 
+/**
+ * @typedef {Object} PropTypes
+ * @static
+ * @prop {string} [className]
+ * @prop {Element} [component] - The element to show as the dropdown
+ * @prop {function} [onClose] - Function to trigger when click happens outside of the DOM element
+ * @prop {boolean} [stayOpen] - Decides if the menu should call the onClose function or not
+ * @prop {Object} [anchorEl] - DOM node to position itself against,
+ * needs to have the `getBoundingClientRect` function on its
+ * `prototype`.
+ */
 DropMenu.propTypes = {
     className: propTypes.string,
-    /** The component to use as the dropdown component */
     component: propTypes.element,
-    /** Function to trigger when click happens outside of the DOM element */
     onClose: propTypes.func,
-    /** Decides if the menu should call the onClose function or not */
     stayOpen: propTypes.bool,
-    /** DOM node to position itself against */
+    /**  */
```

Comment:
Can be removed right?

Reply:
Yeah.

Reply:
https://github.com/dhis2/ui-core/pull/552